### ISSUE TITLE
Radio button for one option doesn't disable entering text in the opposite option, which is then not saved

### ISSUE THREAD

**Issue by lunarovic** [qooob/authentic-theme#1130]
Virtualmin => Backup and Restore => Cloud Storage Providers => Amazon S3 (the same applies to Rackspace)

I filled the two fields "Access key" and "Secret key", but I didn't switch from "None set" to "Entered below ..", so when I clicked "Save", what I entered in those fields was not saved. So somebody could think they saved some important data even though they didn't, and they will later find out and be frustrated. 🙂

So the text fields need to be disabled like elsewhere. A good example is right below those two fields - "S3-compatible server hostname", and if "Use Amazon S3" is selected, the text field next to it is disabled. I suspect there are more instances of this issue somewhere in Webmin, but I can't seem to find them now.

**Comment by qooob** [comment]
That's should be done on server side script as it's always were. 

You mean it's not saved, because user didn't toggle the radio, while textbox wasn't disabled and you could place data in it?

**Comment by lunarovic** [comment]
> You mean it's not saved, because user didn't toggle the radio, while textbox wasn't disabled and you could place data in it?

Yes, exactly. I know it's user's fault, but it's easy to miss, especially because everywhere else the text fields are disabled until you select the radio option for them to enable them, so people are used to knowing that if a text field is enabled for typing, it will save.

**Comment by qooob** [comment]
Does it work differently in Virtualmin Framed Theme?

**Comment by qooob** [comment]
I bet it's very annoying and time consuming to do this on server side for Jamie and Joe, and that's the reason why it's not added everywhere.

I'll try doing it.

**Comment by lunarovic** [comment]
Thanks! But this is merely a pet peeve, not anything of priority.

On a similar note, if I select a dropdown box which is under the second radio button and select an entry, the radio button will not change to that and the first radio button will stay selected. See:

![image](https://user-images.githubusercontent.com/2381553/39396510-c69b62b2-4aef-11e8-9d8e-5b70131bfbba.png)

In modern applications and operating systems, when you do such a thing, it will automatically select the radio button for the option you are changing, because it's obvious that you want to change that option, and not stay with the other radio button.

**Comment by lunarovic** [comment]
I just realized, if the dropdown was disabled, then this couldn't even happen. Same applies to text fields.

**Comment by lunarovic** [comment]
That's all right, thank you for taking the time to look into this!

Best regards
